.tekton/central-db-pull-request.yaml

```diff
@@ -139,7 +139,7 @@ spec:
         1h, 2d, 3w for hours, days, and weeks, respectively.
       name: image-expires-after
       type: string
-    - default: "false"
+    - default: "true"
```

Comment:
Sorry, but I meant not changing the default here but adding a parameter in `spec.params`, lines 22-42. Doing that way should let us more easily see what we customized.

Reply:
Sure, but what do you think about the result of the test?

Reply:
You mean that warning `Image registry.redhat.io/rhel8/postgresql-13:latest does not come from supported registry. Skip handling the sources from it.`? I overlooked it in logs since it has `INFO` on it, noticed only now :-(

I've seen a PR auto-opened with task updates. Not sure if sources task is there. I'd suggest to check if an updated task fixes that. If not, please report it to Slack, but I wouldn't block the PR on it because it's not failing the step.

Reply:
`registry.access.redhat.com` is the only supported registry for the base image: https://github.com/redhat-appstudio/build-definitions/blob/main/containers/source-build/app/source_build.py#L185-L190. 
I'll ask on Slack.